src/shared/capture/captureinterface.cpp

```diff
@@ -46,6 +46,9 @@ void CaptureInterface::readAllParameterValues() {
 }
 
 bool CaptureInterface::copyAndConvertFrame(const RawImage & src, RawImage & target) {
+  if(target.getColorFormat() == COLOR_UNDEFINED)
```

Comment:
Why was this necessary? Maybe add a short comment, explaining it.

Reply:
COLOR_UNDEFINED has a pixel size of 0, which causes a segmentation fault in the following memcpy during the first frame in capture_thread.cpp#327 (`bool bSuccess = capture->copyAndConvertFrame( pic_raw,d->video);`).

Reply:
I still do not understand why your change fixes the issue. The color format should be applied anyways, except if the image is of size zero, in which case the data is equal to `nullptr`.